docs/lang/articles/visualization/ggui.md

```diff
@@ -118,6 +325,55 @@ You can render a scene on a canvas.
 canvas.scene(scene)
 ```
 
+### Fetching Color/Depth information
+
+```python
+img = window.get_image_buffer()
+window.get_depth_buffer(scene_depth)
+depth = window.get_depth_buffer_as_numpy()
+```
+
+After rendering the current scene, you can fetch the color and depth information of the current scene using `get_image_buffer()` and `get_depth_buffer_as_numpy()`, which copy the gpu data to a NumPy array(cpu).
+`get_depth_buffer()` copies the GPU data to a Taichi field (depend on the `arch` you choose) or copies data from GPU to GPU.
+
+:::example
+
+1. Example of fetching color information
+```python
+window = ti.ui.Window("Test for getting image buffer from ggui", (768, 768), vsync=True)
+video_manager = ti.tools.VideoManager("OutputDir")
+
+while window.running:
+    render_scene()
+    img = window.get_image_buffer()
+    video_manager.write_frame(img)
+    window.show()
+
+video_manager.make_video(gif=True, mp4=True)
+```
+
+2. An example of fetching the depth information
+```python
+window_shape = (720, 1080)
+window = ti.ui.Window("Test for copy depth memory", window_shape)
```

Comment:
`depth information` and `depth memory` -> `depth data`

Reply:
ok, I will fix that.